fuzz/fuzz_targets/differential_spec.rs

```diff
@@ -0,0 +1,13 @@
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+use wasmtime_fuzzing::{generators, oracles};
+
+fuzz_target!(|data: (
+    generators::Config,
+    wasm_smith::ConfiguredModule<oracles::SingleFunctionModuleConfig>
+)| {
+    let (config, mut wasm) = data;
+    wasm.ensure_termination(1000);
+    oracles::differential_spec_execution(&wasm.to_bytes()[..], &config);
```

Comment:
The `[..]` shouldn't be required here, since `&Vec<u8>` will coerce to `&[u8]` via automatic deref coercion.